t/009-error-log-append.t

```diff
@@ -0,0 +1,211 @@
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 6) - 6;
+
+#no_diff();
+no_long_string();
+#master_on();
+#workers(2);
+check_accum_error_log();
+run_tests();
+
+__DATA__
+
+
+=== TEST 1: value is appended correctly to error logs
```

Comment:
should we have a "sanity case", which means that if no `lua_kong_error_log_request_id` the error log will not output string "kong_request_id"